lib/DiscoveryStreamFeed.jsm

```diff
@@ -29,6 +29,252 @@ const PREF_SHOW_SPONSORED = "showSponsored";
 const PREF_SPOC_IMPRESSIONS = "discoverystream.spoc.impressions";
 const PREF_REC_IMPRESSIONS = "discoverystream.rec.impressions";
 
+const DEFAULT_LAYOUT_RESP = {
```

Comment:
maybe put it at the bottom so you don't have to scroll as much? :P

Reply:
Like, put the const itself at the bottom of the file?

Reply:
I don't know if I follow.

Reply:
Maybe like how we hide things at the end here:
https://github.com/mozilla/activity-stream/blob/27af57c7499e5735f2a780ef1ad1127cd7687c57/lib/FilterAdult.jsm#L9-L10

Reply:
:+1: